### PR TITLE
Adds manual migration info to the Migration page

### PR DESCRIPTION
**Pull request by momo-the-monster**
We'll need to create, deploy and link the World and UdonSharp Unity Package files described in [this new section](https://63effa4d6938f502e955d708--vrc-udonsharp.netlify.app/migration#manual-migration) before publishing.

### CODE REVIEW

**Review by MerlinVR — APPROVED** [review]
Might want to add a note recommending to backup your project before doing anything, otherwise good. Also do we have an editor flow for creating unitypackages that target the Packages directory? Otherwise I think we'd need to fix the parser that `create-unitypackage` uses because Unity throws errors on importing the meta files and reverts the fixes in #101

**Comment by momo-the-monster** [comment]
@MerlinVR - thanks for the heads-up on that issue with #101, I hadn't seen your comments there yet.
Yes, the Resolver package adds the ability to right-click a package directory in your Project view and select "Export VPM Package", which will make a package named for the id and version of the target package and place it in a new directory called "Exports" at the root of the project.

File: Tools/Docusaurus/docs/Migration.md [hunk 1]
```diff
@@ -54,4 +52,12 @@ UdonSharp 0.x (the .unitypackage version) is deprecated and no longer supported.
...
\ No newline at end of file
+- Obsoleted overloads for station and player join events may no longer be used
+
+## Manual Migration
+
+Follow these steps to upgrade a project that uses a version of UdonSharp below 1.0 without using the Creator Companion:
+
+1. Delete the VRCSDK folder, Udon folder, UdonSharp folder, and Gizmos/UdonSharp folders from your project's "Assets" folder.
+2. Download and install the Unity Package versions of the World SDK from `link coming soon`.
+3. Download and install the Unity Package version of the UdonSharp SDK from `link coming soon`.
```
Comment (Faxmashine) [inline Tools/Docusaurus/docs/Migration.md:63]:
```suggestion
1. Back up your project.
2. Delete the VRCSDK folder, Udon folder, UdonSharp folder, and Gizmos/UdonSharp folders from your project's "Assets" folder.
3. Download and install the Unity Package versions of the World SDK from `link coming soon`.
4. Download and install the Unity Package version of the UdonSharp SDK from `link coming soon`.
```

**Review by Faxmashine — APPROVED** [review]
Looks good.
I agree with @MerlinVR , making 'back up your project' step 1 would be good.
Don't forget to add those links! 😊

**Pull request merged**